src/App.tsx

```diff
@@ -117,14 +121,16 @@ const App = (): JSX.Element => {
       );
       dispatch(isPolkaBtcLoaded(true));
       setIsLoading(false);
-    } catch (error) {
+      // NOTE: Catch clause variable type annotation must be 'any' or 'unknown'. Use of any here
+      // and throughout is to resolve type errors.
+    } catch (error: any) {
       toast.warn('Unable to connect to the BTC-Parachain.');
       console.log('[loadPolkaBTC] error.message => ', error.message);
     }
 
     try {
       startFetchingLiveData(dispatch, store);
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
@tomjeatt 
May I ask about this `any`? I wonder if it should be Error.

Reply:
Actually, I think using `any` is an anti-pattern in TypeScript. I wonder if there's a warning related to using `any`.
OK, as it's not a critical problem, let's do it later on.

Reply:
One question, was there any warning or error with the error argument not being typed?

Reply:
Do you have some opinion on using `Error` instead of `any`?

Reply:
You can't :) if you try you'll get a type error. The issue here isn't really the type annotation, it's the structure of the try/catch functions and how we're handling errors. It's something we should look at, but outsider of the scope of the ticket. The only reason I've typed the errors as `any` is because having it untyped was triggering type errors.

<img width="866" alt="Screenshot 2022-01-21 at 13 21 29" src="https://user-images.githubusercontent.com/40243778/150534039-22540340-47bf-4181-b064-2d6aff21fc8f.png">

Reply:
It makes sense. Sometimes I end up using `any` for narrowing down ticket scopes.